dashboard/test/controllers/api/v1/pd/course_facilitators_controller_test.rb

```diff
@@ -1,27 +1,17 @@
 require 'test_helper'
 
 class Api::V1::Pd::CourseFacilitatorsControllerTest < ::ActionController::TestCase
-  test 'admins can see course facilitators' do
-    sign_in create(:admin)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :success
-  end
-
-  test 'workshop organizers can see course facilitators' do
-    sign_in create(:workshop_organizer)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :success
-  end
-
-  test 'facilitators cannot see course facilitators' do
-    sign_in create(:facilitator)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :forbidden
-  end
-
-  test 'teachers cannot see course facilitators' do
-    sign_in create(:teacher)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :forbidden
+  {
+    admin: :success,
+    workshop_organizer: :success,
+    facilitator: :forbidden,
+    teacher: :forbidden
+  }.each do |user_type, response|
+    test_user_gets_response_for(
+      :index,
+      user: user_type,
+      response: response,
+      params: {course: Pd::Workshop::COURSES.first}
+    )
```

Comment:
Do you mean rename `test_user_gets_response_for`?

It's not always a user type. It can also be a user object. And under the hood it calls `assert_response`, which is a common pattern, so it seems reasonable to use `response` over `http_status_code`. I also like ending in `_for` with the first argument as the action.